backoff.py

```diff
@@ -281,7 +292,15 @@ def retry(*args, **kwargs):
                                   'value': ret})
                         break
 
-                    seconds = next(wait) + jitter()
+                    try:
+                        if jitter is not None:
+                            seconds = jitter(next(wait))
```

Comment:
Does it looks below?

    value = next(wait)
    try:
        if jitter is not None:
            seconds = jitter(value)